vlsi/example-vlsi

```diff
@@ -50,16 +54,21 @@ class ExampleDriver(CLIDriver):
             # Default set of steps can be found in the CAD tool plugin's __init__.py
 
             # make_pre_insertion_hook will execute the custom hook before the specified step
-            hammer_vlsi.HammerTool.make_pre_insertion_hook("route_design", example_add_fillers),  # SYNTAX: make_pre_insertion_hook("EXISTING_STEP", INSERTED_HOOK)
+            # SYNTAX: make_pre_insertion_hook("EXISTING_STEP", INSERTED_HOOK)
+            # hammer_vlsi.HammerTool.make_pre_insertion_hook("route_design", example_add_fillers),
+
             # make_post_insertion_hook will execute the custom hook after the specified step
             hammer_vlsi.HammerTool.make_post_insertion_hook("init_design", example_tool_settings),
+
             # make_replacement_hook will replace the specified step with a custom hook
-            hammer_vlsi.HammerTool.make_replacement_hook("place_tap_cells", example_place_tap_cells),
+            # hammer_vlsi.HammerTool.make_replacement_hook("place_tap_cells", example_place_tap_cells),
```

Comment:
Its unclear which of these hooks is just here to show syntax and which are actually needed.

Reply:
Ok, I'll move the unused ones that are showing syntax to a different section in the list.